### ISSUE TITLE
Unable to generate a ninjaIndex file 

### ISSUE THREAD

**Issue by RChGO** [FischbachLab/ninjaMap#2]
Hi! I encountered an issue when using the command "nextflow run /share/data2/guorc/Software/ninjaMap-master/nf-core-ninjaindex/main.nf --genomes 'genome/*.fasta' --outdir 'db_index'". Every time I run it, the first three steps complete successfully, but when it reaches the fourth step, it consistently reports the following error. I'm not sure how to resolve this problem; I was wondering if you might have any insights on how to fix it? Could you please provide some input files for us to use the command and conduct testing?

![1690450512253](https://github.com/FischbachLab/ninjaMap/assets/45479925/23cc8bf2-7489-491f-bb5a-a5f651820a69)
![1690450617008](https://github.com/FischbachLab/ninjaMap/assets/45479925/2ee71354-a59f-4997-9210-6443729de862)

**Comment by xmeng** [comment]
Please try the latest version at https://github.com/FischbachLab/nf-ninjaindex and let me know if it works at your side.

**Comment by RChGO** [comment]
> Please try the latest version at https://github.com/FischbachLab/nf-ninjaindex and let me know if it works at your side.


Yes, I have tried the latest version, but the same issue persists. Could it be because there are forbidden characters in the contig names within my input files?

**Comment by RChGO** [comment]
> > Please try the latest version at https://github.com/FischbachLab/nf-ninjaindex and let me know if it works at your side.
> 
> Yes, I have tried the latest version, but the same issue persists. Could it be because there are forbidden characters in the contig names within my input files?

Hello! I have resolved this issue. I noticed that there are one problems when running "ninjaIndex_multiBams.sh," which result in the failure to recognize files with a suffix other than ".fna". In lines 59 to 62 of the script "ninjaIndex_multiBams.sh," it appears that the "-ext" parameter is not being called (as shown in Figure 1), which means only files with the ".fna" extension can be recognized. Would it be better to consider adding the call for this parameter?

**Figure1**
![1690528342245](https://github.com/FischbachLab/ninjaMap/assets/45479925/d9d1427b-4695-4340-af15-056adc6f3a83)
